src/GitHub.InlineReviews/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
```

Comment:
This `source.extension.vsixmanifest` shouldn't be used (we're not creating a .vsix for this project). Can we delete it from this project?  The same would apply to the on in `GitHub.StartPage`.

Am I missing something and there's a reason why they can't be deleted?

Reply:
The vsix file has an entry exporting the pkgdef. I'm not sure you can kill the manifest and still have the project packaging properly. I think you may have to change the project type, and/or maybe have another project type for the pkgdef-but-no-vsix logic. There used to be a project type for this at some point iirc, but honestly I can't be sure so it would need testing.

Reply:
That's what I thought as well, but I tried deleting the `.vsixmanifest` and it still seemed to compile. Maybe this has changed in a recent VSIX project type update? Dunno.

Reply:
Could have, I wouldn't be surprised. I remember the file being mandatory but if they updated the targets file to skip it since then, then that would make sense.